perf/benchmark/configs/istio/security_authz/latency.yaml

```diff
@@ -0,0 +1,18 @@
+# Data: latency
+# Config: security_authz
+telemetry_mode: "none"
+conn:
+    - 2
```

Comment:
should we also want to test other `conn` and `qps`?

Reply:
we can try with a small set of config first